modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/utilities/modules.js

```diff
@@ -12,32 +12,54 @@
  * details.
  */
 
+import React from 'react';
+
 export function getLiferayJsModule(moduleUrl) {
 	return new Promise((resolve, reject) => {
 		Liferay.Loader.require(
 			moduleUrl,
-			(jsModule) => {
-				return resolve(jsModule.default || jsModule);
-			},
-			(err) => {
-				return reject(err);
-			}
+			(jsModule) => resolve(jsModule.default || jsModule),
+			(err) => reject(err)
```

Comment:
nit: `err` could be `error`

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/112/commits/3f8c39bca24ec25d08eb18896e159c270a8a2922 :)